### PR TITLE
mention discard mount option

### PR DESCRIPTION
**Pull request by awokd**
nothing 4.0 specific

### CODE REVIEW

File: common-tasks/tips-and-tricks.md [hunk 1]
```diff
@@ -58,4 +58,5 @@ Trim for standalone AppVMs
 ---------------------
 The `qvm-trim-template` command is not available for a standalone AppVM.
 
-It is still possible to trim the AppVM disks by using the `fstrim --all` command from the appvm
+It is still possible to trim the AppVM disks by using the `fstrim --all` command from the appvm.
+Note you may need to add the `discard` option to the mount line in `/etc/fstab` inside the standalone AppVM before this will work as expected.
```
Comment (marmarek) [inline common-tasks/tips-and-tricks.md]: Those two things are not related in the way you've described:
 - `fstrim` manually send discard requests for unused space
 - `discard` option automatically send discard requests whenever space is freed (file delete, etc)

**Comment by awokd** [comment]
Oh good point. I was thinking of LUKS. I'll clarify.
